IFC4x3/PropertyConstants/p/PDH_14Lj$vkD1BcvoagHoMZdwM.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PDH_14LjvkD1BcvoagHoMZdwM" Name="PDH" UniqueId="4456dff9-b8d0-4b9b-9ca4-a91c968e7e96">
+	<Documentation>PDH transport equipment.</Documentation>
```

Comment:
Don't know if it helps to provide the meaning of the acronym (plesiochronous digital hierarchy from googling) :-)